README.md

```diff
@@ -1,2 +1,186 @@
 # react-simple-scroll
```

Comment:
Good practice for any open source frontend project would be to document browser support guarantees